api/profile/profileRouter.js

```diff
@@ -292,4 +292,44 @@ router.delete('/:id', (req, res) => {
   }
 });
 
+/**
+ * @swagger
+ * /profile/{id}/children:
+ *  get:
+ *    summary: To fetch children for each profile
+ *    description : Get the list of children for a particular profile id
+ *    security:
+ *      - okta: []
+ *    tags:
+ *      - profile
+ *    parameters:
+ *      - $ref: '#/components/parameters/profileId'
+ *    responses:
+ *      404:
+ *        $ref: '#/components/responses/NotFound'
+ *      200:
+ *        description: A profile object
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *                message:
+ *                  type: string
+ *                  description: A message about the result
+ *                  example: Profile id found.
+ *                profile:
+ *                  $ref: '#/components/schemas/Profile'
+ */
+router.get('/:id/children', async (req, res) => {
+  const id = req.params.id;
+  Profiles.findChildren(id)
+    .then((children) => {
+      res.status(200).json(children);
```

Comment:
You might need to create middleware here to see if the profile id exists.  

and/ or an if-else statement to test if the array being sent back has more than a length of 0.  Right now if I test this endpoint with a profile id that does not exist it returns a status of 200.  This is because children is an array of objects and an empty array will pass 200 status.

Reply:
Solved by adding crudOperations